### ISSUE TITLE
fetch_model's use of log levels makes it die with unconfigured logger

### ISSUE THREAD

**Issue by sorenmulli** [peleiden/daluke#107]
hPelutils version 0.6.7
To reproduce:
```py
from daluke.api.fetch_model import fetch_model, Models
fetch_model(Models.DaLUKE)
```
Error:
```py
~/Nextcloud/bach/daLUKE/daluke/api/fetch_model.py in fetch_model(model, force_download)
     55     # This is done in a seperate working directory
     56     log.debug("Loading entity vocab, metadata, and state dict")
---> 57     with log.level(len(Levels)+1):
     58         cwd = os.getcwd()
     59         os.chdir(_download_dir)

~/.pyenv/versions/luke38/lib/python3.8/site-packages/pelutils/logger.py in level(self, level)
    180 
    181     def level(self, level: Levels):
--> 182         return self._level_mgr.with_level(level)
    183 
    184     @property

~/.pyenv/versions/luke38/lib/python3.8/site-packages/pelutils/logger.py in _level_mgr(self)
    115     @property
    116     def _level_mgr(self) -> _LevelManager:
--> 117         return self._logger["level_mgr"]
    118     @property
    119     def _level(self) -> Levels:

KeyError: 'level_mgr'

```

This needs to be fixed for #103 to be done as this bug requires the user to configure a logger for API interaction.
I don't know if this is to be fixed here or in pelutils, or if we should just remove calls to logger from within fetch_model and predict_mlm and predict_ner

**Comment by sorenmulli** [comment]
Thoughts, @asgerius? (NB: working on API at `feature/api`)

**Comment by asgerius** [comment]
Thinking we should just remove logger calls. Also ugly to have an api print out stuff

**Comment by asgerius** [comment]
I think we should remove them to prevent printing in future pelutils versions

**Comment by sorenmulli** [comment]
Then we should remove in other functions that are called by API, and, besides they all use DEBUG, so they would not print in future version.

**Comment by asgerius** [comment]
Yes, they should be removed.
Also, we'll apparently have to discuss this, but I definately think unconfigured loggers should print debug

**Comment by sorenmulli** [comment]
Can look at a solution tomorrow, IMO it would be a shame to remove  things like the serialize functions telling whether they uses system or built-in tar module only for this. Should probably then find a way to suppress their output that works for unconfigures logger  from within fetch_model

**Comment by sorenmulli** [comment]
Made new issue for no-logging: #109, as this specific buggy is fixed in 7e4e738
